src/Window.cpp

```diff
@@ -26,17 +26,13 @@ int start(int argc, char *argv[])
 Window::Window(QWidget* parent)
     : QWidget(parent)
 {
-  this->game_start_movie = new QMovie(":/img/game_start_hover.gif");
-  this->game_start_movie->start();
-  this->game_start_movie->setPaused(true);
-
-  _masterFrameUI.setupUi(this->master_frame);
+  this->_gameStartMovie = new QMovie(":/img/game_start_hover.gif");
+  this->_gameStartMovie->start();
+  this->_gameStartMovie->setPaused(true);
 
-  //auto start_button = new QLabel(this->master_frame);
-  //start_button->setGeometry(this->_masterFrameUI.l_game_start_frame->geometry());
-  //start_button->setMovie(game_start_movie);
+  _masterFrameUI.setupUi(this->_masterFrame);
```

Comment:
Be consistent. Either use `this->` access everywhere, or nowhere. It is recommended to not use them, because they make lines longer than they have to be + it's more work for nothing. 

We differentiate between local variables and class members with the "_" prefix.